### PR TITLE
Add year, bit and varbit types

### PR DESCRIPTION
**Pull request by karatakis**
## PR Info
There are many non standard SQL types that we haven't covered yet. I tried to add some.

I need some of them to close this issue
https://github.com/SeaQL/seaography/issues/52
And cover some of FIXME tasks
https://github.com/SeaQL/sea-schema/blob/master/src/mysql/writer/column.rs

## Adds
* [x] YEAR MySQL
* [x] BIT MySQL, Postgres
* [x] VARBIT MySQL Postgres

### CODE REVIEW

**Review by ikrivosheev — COMMENTED** [review]
@karatakis hello! Thank you for PR, can you rustfmt?

**Comment by karatakis** [comment]
@ikrivosheev done

**Review by ikrivosheev — APPROVED** [review]
@karatakis thank you! LGMT!

File: src/table/column.rs [hunk 1]
```diff
@@ -81,6 +84,13 @@ pub enum PgInterval {
     MinuteToSecond,
 }
 
+// All MySQL year type field length sizes
+#[derive(Debug, Clone)]
+pub enum MySQLYear {
```
Comment (tyt2y3) [inline src/table/column.rs:89]: My only wish is to name it as `MySqlYear`.

**Comment by ikrivosheev** [comment]
@karatakis can you create PR in https://github.com/SeaQL/sea-schema/?

**Comment by karatakis** [comment]
@ikrivosheev I will start ASAP

**Pull request merged**